pkg/scyllaclient/client_scylla.go

```diff
@@ -314,6 +314,22 @@ func (c *Client) KeyspacesByType(ctx context.Context) (map[KeyspaceType][]string
 	return out, nil
 }
 
+// AllTables returns all tables grouped by keyspace.
+func (c *Client) AllTables(ctx context.Context) (map[string][]string, error) {
+	resp, err := c.scyllaOps.ColumnFamilyNameGet(&operations.ColumnFamilyNameGetParams{Context: ctx})
+	if err != nil {
+		return nil, err
+	}
+	out := make(map[string][]string)
+	for _, kst := range resp.Payload {
+		parts := strings.Split(kst, ":")
```

Comment:
Could you please add simple validation checking if the len(parts) > 1 ?
It's to avoid the situation that scylla manager panics when (somehow) the response is not ...:...